end_to_end/fraud_detection/pipeline-e2e.ipynb

```diff
@@ -4,32 +4,37 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    start_response.wait(delay=60, max_attempts=500)
</pre><p>Is 500 attempts too many? What's the cadence, like how many minutes would 500 attempts take?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>It is needed, otherwise the notebook will complain that we did not wait sufficiently long enough and the pipeline will be interrupted, causing CI errors. Initially, I thought the cadence is the delay, but from the CI behavior, this doesn't seem to be the case. </p>